specification/sbi_atee.adoc

```diff
@@ -540,12 +540,6 @@ struct tvm_create_params {
      * returned by the call to sbi_tee_host_get_tsm_info().
      */
     unsigned long tvm_state_addr;
-    /*
-     * The vcpuid for the VCPU that will be designated as the boot VCPU.
-     * The host must add create a VCPU with this vcpuid by calling `sbi_tee_host_create_tvm_vcpu`
-     * before calling `sbi_tee_host_finalize_tvm().
-     */
-    unsigned long tvm_boot_vcpuid;
```

Comment:
I think we put this in here with the intention of implementing it some day. But if you all don't think it's important to be able to select a non-0 vCPU as the boot vCPU, then that's fine with me. I think we should spec that the boot vCPU is vCPU 0 in that case, instead of making it implementation-defined, though.

Reply:
I am not able to think of any scenario where host might want to boot a particular VCPU first. Fixing boot VCPU to `0` sound good to me.